ui/StatusQ/src/typesregistration.cpp

```diff
@@ -40,6 +40,10 @@
 #include "onboarding/enums.h"
 
 void registerStatusQTypes() {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    qputenv("QT_FILE_SELECTORS", QByteArrayLiteral("qt6,") + qgetenv("QT_FILE_SELECTORS"));
```

Comment:
If `QT_FILE_SELECTORS` is empty, then you'll get `qt6,` as the result. Maybe not critical but worth fixing imo

Reply:
Sth like:
```suggestion
    QStringList selectors = qgetenv("QT_FILE_SELECTORS").split(',');
    selectors << "qt6";
    qputenv("QT_FILE_SELECTORS", selectors.join(","));
```

Reply:
Or check first for `qEnvironmentVariableIsEmpty()`

Reply:
Fixed